From edb89919e357a849cc15227944b7ef75fbcbf43c Mon Sep 17 00:00:00 2001
From: abelino <abelino.romo@gmail.com>
Date: Sat, 3 Aug 2024 19:58:19 -0700
Subject: [PATCH] Prefer SYS_MMC_ENV_PART config over using default env value

---
 board/compulab/common/mmc.c | 21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/board/compulab/common/mmc.c b/board/compulab/common/mmc.c
index b2922122b2..b210017972 100644
--- a/board/compulab/common/mmc.c
+++ b/board/compulab/common/mmc.c
@@ -33,24 +33,3 @@ int board_mmc_get_env_dev(int devno)
 	env_dev = _board_mmc_get_env_dev(devno);
 	return env_dev;
 }
-
-static int _mmc_get_env_part(struct mmc *mmc)
-{
-	const ulong user_env_part = env_get_hex("env_part", ULONG_MAX);
-	if (user_env_part != ULONG_MAX) {
-		printf("User Environment part# is (%lu)\n", user_env_part);
-		return (int)user_env_part;
-	}
-
-	return EXT_CSD_EXTRACT_BOOT_PART(mmc->part_config);
-}
-
-uint mmc_get_env_part(struct mmc *mmc)
-{
-	if (mmc->part_support && mmc->part_config != MMCPART_NOAVAILABLE) {
-		uint partno = _mmc_get_env_part(mmc);
-		env_part = partno;
-		return partno;
-	}
-	return 0;
-}
--
2.46.0
